examples/python/imu_signals/README.md

```diff
@@ -3,6 +3,8 @@ title = "IMU signals"
 tags = ["Plots"]
 description = "Log multi dimensional signals under a single entity."
 thumbnail_dimensions = [480, 480]
+channel = "nightly"
```

Comment:
as mentioned on our call this morning, this makes it show up only for Rerun devs essentially. But I really think this should show in shipped versions of Rerun on the welcome page :)
Therefore either `main` or if its slow building `release` should be here

Reply:
In which case, it should also be in manifest.toml, such that it shows up on our landing page.

Reply:
yesplease! forgot about manifest.toml

Reply:
it's already in the manifest.toml 😊